app/assets/stylesheets/components/_forms.scss

```diff
@@ -67,27 +63,35 @@
     gap: 0.5rem;
     grid-template-columns: 1fr 1fr 1fr;
   }
+}
 
-  &__errors {
-    display: grid;
-    gap: 0.5rem;
-    list-style: none;
-    margin-top: 1rem;
+.newForm__title {
+  text-align: center;
+}
```

Comment:
Regarding these classes called **newForm**, what do you mean by that name? Remember that the more defined the name of your class, the better it is within your development.

Reply:
those classes used to be inside the .newform but to reuse them in the new forms I decide to move them outside of the .newform